master/conform.html

```diff
@@ -368,28 +368,6 @@ <h2 id="referencing-modes">Processing modes for SVG sub-resource documents</h2>
       [<a href="refs.html#ref-html">HTML</a>]</p>
     </dd>
 
-    <dt id="embedded-document-mode"><a>'iframe'</a> references</dt>
-    <dd>
-      <p>SVG documents referenced by the
-      an HTML <a>'iframe'</a> element in an SVG document must use the
-      same processing mode as the embedding document,
-      subject to any restrictions defined by the
-      <a href="https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-iframe-sandbox"><span class="attr-name">sandbox</span> attribute</a>
-      on the embedding <a>'iframe'</a>.</p>
-
-      <p class="note">The same processing rules are intended to be used when
```

Comment:
We need to preserve the recommendations in this note, even without `<iframe>` in an SVG document.

Reply:
We intended to add the annotation to make it clear for later edits why a certain feature was added or changed (maybe even removed) from the previous REC. In this case the feature weren't present in SVG 1.1 and dropped in SVG 2.0. Why do we want to keep the annotation?

Reply:
Processing modes in general are new in SVG 2. We have explicit requirements for processing modes for SVG embedded in other SVG, but we also have these notes with recommendations for processing modes for SVG embedded in HTML (which match what browsers, etc. do, but aren't explicitly spec'd anywhere).

Maybe there should be a separate sub-section, and the note about HTML images can be moved there, too?

Reply:
Agree with Amelia that discussion of processing modes should be retained.

Reply:
> Maybe there should be a separate sub-section, and the note about HTML images can be moved there, too?

In the interests of moving forward, I believe we should accept the PR and then add the separate subsection in a different PR.